sdk/src/test/java/com/mapbox/maps/NativeMapTest.kt

```diff
@@ -16,34 +16,22 @@ class NativeMapTest {
   fun subscribe() {
     val nativeMap = NativeMapImpl(map)
     val list = mutableListOf("foobar")
-    nativeMap.subscribe(
-      object : Observer() {
-        override fun notify(event: Event) {}
-      },
-      list
-    )
+    nativeMap.subscribe({ }, list)
```

Comment:
nit:
```suggestion
    nativeMap.subscribe({}, list)
```

Reply:
`ktlint` is passing so let's leave as is, I see no difference tbh 😄